### ISSUE TITLE
Uninstall application

### ISSUE THREAD

**Issue by SimonLab** [dwyl/github-backup#87]
Users can remove/uninstall a Github app from their repositories.
In thie scenario should we also delete all the backup data from our server?
I would suggest that we don't do this automatically as if a Github app is uninstall by mistake the user won't be able to recover their backup history. Instead we can maybe send an email to ask the users to confirm that they want to delete all their backups?
@nelsonic @iteles @Cleop do you have any ideas on this?

**Comment by nelsonic** [comment]
@SimonLab agree. we do _not_ want to _automatically_ `delete` the issue/comment history.
Sending an email to the person who _originally_ created the app would be _good_.
However we _may_ run into the issue that the _original_ person has _left_ the organisation 
so sending them an email will "bounce" ... 
a consideration for _later_; for now, sending an email with a link and unique token would be good.
I don't think of this as "MVP", but if you _want_ to work on it, go for it.
